Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM registry.hub.docker.com/library/debian:stable-slim
+
+LABEL Maintainer="r.siudak@ultimaker.com" \
+      Comment="Ultimaker python3-inotify environment"
+
+RUN apt-get update && \
+    apt-get install -y \
+    python3-stdeb \
+    python-all
```

Comment:
You can get a smaller package by removing the package list afterward, this also increases the likelihood that the layer can be cached.

 && rm -r /var/lib/apt/lists/*

Reply:
also apt-get clean && \ (or rm -rf /var/cache/apt)